manager/controllers/app/blueprint_controller_test.go

```diff
@@ -82,6 +82,57 @@ func TestBlueprintReconcile(t *testing.T) {
 	g.Expect(blueprint.Status.Releases).Should(gomega.HaveKeyWithValue("notebook-default-notebook-read-module", blueprint.Status.ObservedGeneration))
 }
 
+// Tries to create a blueprint in a namespace other than m4d-blueprints.
+// Result should be that no event is sent to the reconciler, since the predicate function
+// registered during the controller deployment filters it out.
+func TestIllegalBlueprint(t *testing.T) {
+	t.Parallel()
+	g := gomega.NewGomegaWithT(t)
+	logf.SetLogger(zap.New(zap.UseDevMode(true)))
+
+	blueprint, err := readBlueprint("../../testdata/blueprint.yaml")
```

Comment:
Don't you need to set namespace to `rogue-namespace` here so the right value would be used in `objs` (line 97) too?

Reply:
@roee88 This line reads the blueprint spec from the yaml.  The spec doesn't include the namespace.  The namespace is supplied with the reconcile request.  See line 121.

Reply:
Moved to integration tests